### ISSUE TITLE
It's working great! No errors, keep it up. 👍😜🏆

### ISSUE THREAD

**Issue by zhouxinghong** [chen08209/FlClash#635]
It's working great! No errors, keep it up. 👍😜🏆

**Comment by xianhongtao** [comment]
Bull shit, v0.8.70 is not working on my comptuer

**Comment by xianhongtao** [comment]
I have to roll back

**Comment by zhouxinghong** [comment]
You can throw away your computer.

**Comment by xianhongtao** [comment]
Yes. Let everyone throw away their computers just solve the problem.

**Comment by xianhongtao** [comment]
No user no problems.

**Comment by zhouxinghong** [comment]
![image](https://github.com/user-attachments/assets/98c664a2-8750-4267-8a51-522667e0dcfd)

**Comment by hisbalah** [comment]
works, but where to get profiles.  One yaml have all proxies timeout, another free.yaml give me all time same ip addres no matter on which i click.  so how? or where. thank you guys.

**Comment by xianhongtao** [comment]
> works, but where to get profiles. One yaml have all proxies timeout, another free.yaml give me all time same ip addres no matter on which i click. so how? or where. thank you guys.

Buy VPN subscriptions.
